website/docs/guides/migration/versions/upgrading-to-v1.2.md

```diff
@@ -22,5 +22,6 @@ We added a collection of ["cross-database macros"](cross-database-macros) to dbt
 ## New and changed documentation
 
 _Under construction_
+- Updated `metrics.md` to reflect the addition of the `metric` function and support for `expression` metrics.
```

Comment:
Two things:
- The intent here is not to keep a pure changelog, but rather to give users a "highlight reel" of what's changed, and links to read more
- You'll need to resolve a merge conflict before merging (most PRs for v1.2 target updates to this page)
```suggestion
- [**Metrics**](building-a-dbt-project/metrics) now support an `expression` type (metrics-on-metrics), as well as a `metric()` function to use when referencing metrics from within models or `expression`-type metrics.
```